src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/ad4734fe-5b10-4712-9ec2-a542ee3bb131)

Reply:
not fixed